src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Denial of service.Image Parsing
```powershell
Expected the actual value to be greater than 10, but got 0.404657221570738.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 1897
1897:             $pathologicalRatio | Should -BeGreaterThan 10
```